_applications/var.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Value-at-risk notebook
+link: var
+layout: application
+menu_template: menu_tutorial_application.html
+description: |-
+ The value-at-risk notebook is a simple example of how to run Jupyter notebooks on OpenShift, Monte Carlo simulations in Spark, and how to interactively explore data to find better ways to model it.
+project_links:
+- https://github.com/willb/var-notebook
+---
+
+<h1 id="introduction">Introduction</h1>
+
+This application characterizes historical returns of stock data and then simulates many possible outcomes for a given portfolio over a given time horizon.  The idea is that you can characterize the value at risk for a given percentage of simulated outcomes (e.g., "there is a 5% chance we'd lose more than $1M USD in the next two weeks").  While you wouldn't want to use the techniques in this application to guide real investment decisions, it also demonstrates how to use a Jupyter notebook with Apache Spark in OpenShift and you can modify the basic simulation to model security returns in a more sophisticated manner.
+
+<h1 id="architecture">Architecture</h1>
+
+The driver for the value-at-risk application is a [Jupyter](http://jupyter.org/) notebook.  This notebook connects to a Spark cluster and schedules jobs to perform simulations.
+
+<h1 id="installation">Installation</h1>
+
+Installing the value-at-risk notebook is straightforward; you simply need to crete an OpenShift project, deploy a Spark cluster in that project, and install and run the notebook image.
+
+## Prerequisites
+
+The easiest way to get a Spark cluster going in your OpenShift project is to use [`oshinko-deploy.sh`](https://github.com/radanalyticsio/oshinko-rest/blob/master/tools/oshinko-deploy.sh).  This provides a streamlined installation for the Oshinko cluster manager into your project.  Oshinko is a service to manage Spark clusters for OpenShift projects.
```

Comment:
unless we need the rest server, we should probably switch this link to point at the oshinko-webui repository. it has a separate oshinko-deploy.sh which is functionally equivalent to the rest one, but uses a webui with the integrated cli tooling.